CONTRIBUTING.md

```diff
@@ -8,7 +8,8 @@ Cisco Network Elements support a rich set of features to make networks robust, e
 
 ## Making Changes
 
-* Fork the repository
+* Fork and clone the repository
+  * Run the `bin/git/update-hooks` script to install our recommended Git hooks into your local repository.
```

Comment:
Correct. All of these are client-side hooks (github doesn't permit server-side hooks) so all they can do is assist with best practices, not force anything absolutely.